src/app/main/component/events/components/create-edit-events/create-edit-events.component.ts

```diff
@@ -162,6 +162,12 @@ export class CreateEditEventsComponent extends FormBaseComponent implements OnIn
 
   public checkTab(tag: TagObj): void {
     tag.isActive = !tag.isActive;
+    const tagsArr: Array<string> = this.tags.filter((t) => t.isActive).reduce((ac, cur) => [...ac, cur.nameEn], []);
+    if (tagsArr.length === 0) {
```

Comment:
you can write just `this.checkAfterSend = tagsArr.length !== 0` or `this.checkAfterSend = !!tagsArr.length`

Reply:
Good approach! I'll use it :)